CMakeLists.txt

```diff
@@ -136,7 +136,7 @@ elseif(BUILD_MPS)
                 VERBATIM)
     add_custom_target(metallib DEPENDS "bitsandbytes/bitsandbytes.metallib")
 else()
-    set(LIBSUFFIX "cpu")
+    string(APPEND BNB_OUTPUT_NAME "_cpu")
```

Comment:
👍 Looks like `LIBSUFFIX` isn't used (unless it's a CMake thing I don't know about).

Reply:
No, this was a merge error made by me the other day. I used LIBSUFFIX earlier, this was changed to BNB_OUTPUT_NAME, I missed this line